e2e/tests/attestations/attestations_test.go

```diff
@@ -0,0 +1,891 @@
+	ibcexported "github.com/cosmos/ibc-go/v10/modules/core/exported"
+	attestations "github.com/cosmos/ibc-go/v10/modules/light-clients/10-attestations"
+	ibctesting "github.com/cosmos/ibc-go/v10/testing"
+)
+
+func TestAttestationsTestSuite(t *testing.T) {
+	testifysuite.Run(t, new(AttestationsTestSuite))
+}
+
+type AttestationsTestSuite struct {
+	testsuite.E2ETestSuite
+	attestorKeys []*ecdsa.PrivateKey
+}
+
+func (s *AttestationsTestSuite) SetupSuite() {
+	s.SetupChains(context.TODO(), 1, nil)
+	s.setupAttestors()
+}
+
+func (s *AttestationsTestSuite) setupAttestors() {
+	for range 3 {
+		privateKey, err := crypto.GenerateKey()
+		s.Require().NoError(err)
+		s.attestorKeys = append(s.attestorKeys, privateKey)
+	}
+}
+
+func (s *AttestationsTestSuite) getAttestorAddresses() []string {
+	var addresses []string
+	for _, key := range s.attestorKeys {
+		address := crypto.PubkeyToAddress(key.PublicKey)
+		addresses = append(addresses, address.Hex())
+	}
+	return addresses
+}
+
+func (s *AttestationsTestSuite) signAttestationData(data []byte) [][]byte {
+	hash := sha256.Sum256(data)
+	var signatures [][]byte
+	for _, key := range s.attestorKeys {
+		sig, err := crypto.Sign(hash[:], key)
+		s.Require().NoError(err)
+		signatures = append(signatures, sig)
+	}
+	return signatures
+}
+
+func (s *AttestationsTestSuite) createPacketAttestationProof(height uint64, path []byte, commitment []byte) []byte {
+	packetAttestation := &attestations.PacketAttestation{
+		Height: height,
+		Packets: []attestations.PacketCompact{
+			{
+				Path:       path,
+				Commitment: commitment,
+			},
+		},
+	}
+	attestationData, err := attestations.ABIEncodePacketAttestation(packetAttestation)
+	s.Require().NoError(err)
+
+	signatures := s.signAttestationData(attestationData)
+	proof := &attestations.AttestationProof{
+		AttestationData: attestationData,
+		Signatures:      signatures,
+	}
+	proofBz, err := proto.Marshal(proof)
+	s.Require().NoError(err)
+	return proofBz
+}
+
+func (s *AttestationsTestSuite) createNonMembershipProof(height uint64, path []byte) []byte {
+	zeroCommitment := make([]byte, 32)
+	packetAttestation := &attestations.PacketAttestation{
+		Height: height,
+		Packets: []attestations.PacketCompact{
+			{
+				Path:       path,
+				Commitment: zeroCommitment,
+			},
+		},
+	}
+	attestationData, err := attestations.ABIEncodePacketAttestation(packetAttestation)
+	s.Require().NoError(err)
+
+	signatures := s.signAttestationData(attestationData)
+	proof := &attestations.AttestationProof{
+		AttestationData: attestationData,
+		Signatures:      signatures,
+	}
+	proofBz, err := proto.Marshal(proof)
+	s.Require().NoError(err)
+	return proofBz
+}
+
+func (*AttestationsTestSuite) hashPath(path []byte) []byte {
+	return crypto.Keccak256(path)
+}
+
+func (s *AttestationsTestSuite) prefixedPath(key []byte) []byte {
+	prefixedPath := bytes.Join([][]byte{[]byte(ibcexported.StoreKey), key}, []byte("/"))
+	return s.hashPath(prefixedPath)
+}
```

Comment:
I thought we always use `[][]byte{[]byte("")}` as the counterparty prefix for attestor LC?